Rubberduck.Parsing/Symbols/Attributes.cs

```diff
@@ -92,30 +75,40 @@ public override int GetHashCode()
     /// </remarks>
     public class Attributes : HashSet<AttributeNode>
     {
-        public bool HasAttributeFor(AnnotationType annotationType, string memberName = null)
+        public bool HasAttributeFor(IAttributeAnnotation annotation, string memberName = null)
         {
-            if (!annotationType.HasFlag(AnnotationType.Attribute))
+            if (!annotation.AnnotationType.HasFlag(AnnotationType.Attribute))
             {
                 return false;
             }
 
-            var name = "VB_" + annotationType;
+            var attributeName = memberName != null
```

Comment:
`var attributeName = !string.IsNullOrEmpty(memberName)` might be better here - I'm assuming `.Attribute` wouldn't be legit.

Reply:
Hm, if you add the attribute with an empty member name it is illegal, which means that the VBE will ignore it on reimport.

On the other hand, also not having the dot makes it a module attribute, no matter where it is in the module.

Do we want it to be a module attribute when an empty but not null member name gets passed in?

Reply:
TBH, I don't really think it matters much either way in that the attribute is malformed. I may also be misreading this, but wouldn't this only be possible if it was encountered outside of a module body but not in the module declarations?

Reply:
For `VB_Description` the attribute would not be malformed without the dot. For this, the attribute name is the same for members and modules. The only difference is the `MemberName.` in front for members. The VBE will interpret `VB_Description` (without the member part) as a module description anywhere in a module.